extension/content/customPersonaEditor.js

```diff
@@ -263,6 +263,9 @@ var CustomPersonaEditor = {
       case PersonaService.FIREFOX_ID:
  	window.close();
         break;
+      case PersonaService.SEAMONKEY_ID:
+    window.close();
```

Comment:
Andreas, if I prettify just the **3** lines I added, it does not align with above case and break statements. Because that **Switch** statement is already bad formatted itself.

I just tried to align with the already bad formatted lines.

If you want I can prettify all Switch statement but it will increase modified lines unnecessarily.